extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -1054,6 +1055,7 @@ static public function doPost( $text, $user, $title, $parentId = false, $metadat
 		$article = new Article( $commentTitle, 0 );
 
 		$retVal = self::doSaveAsArticle( $text, $article, $user, $metadata );
+		self::addCommentMapping( $article->getID(), $title->getArticleID(), $parentId ?: 0 ); // SUS-3433
```

Comment:
this code gets called also for Wall/Forum, we should probably put it into doAfterPost method which won't get called for those after https://github.com/Wikia/app/pull/15473